dockerfiles/cleanup.sh

```diff
@@ -1 +1,2 @@
 chown --reference=/data/code -R .
+rm -rf /data/code/Gemfile.lock
```

Comment:
Surely we don't need -rf here? Gemfile.lock should never be a directory or in need of forcing...